app/views/course/group/group_categories/index.html.slim

```diff
@@ -0,0 +1,8 @@
+= page_header do
+  - if can?(:create, Course::GroupCategory.new(course: current_course))
+    div.pull-right
+      div.new-btn
+
+= display_groups_tabs
```

Comment:
Just wondering - why the tabs are not displayed in react instead?

Reply:
Haha can is can, but I personally like the look of the bootstrap tabs over the MUI tabs.

I can also code it out in React (since I just need to apply the bootstrap classes), but I'm not sure if that'd be creating problems in the future when stripping out bootstrap.